content/millennium/dstu2/scheduling/appointment.md

```diff
@@ -55,3 +55,86 @@ List an individual Appointment by its id:
 
 <%= headers status: 200 %>
 <%= json(:dstu2_appointment_read) %>
+
+### Errors
+
```

Comment:
Please apply the new template to the other existing operations as well.

Reply:
Addressed in https://github.com/cerner/fhir.cerner.com/pull/153/commits/06d8cdcf08dbdc4ba1015f4c38a094c527a30253